ui/src/app/shared/components/graph/graph-panel.tsx

```diff
@@ -151,7 +151,7 @@ export const GraphPanel = (props: Props) => {
                     </div>
                 </div>
             )}
-            <div className={'graph ' + props.classNames}>
+            <div className={'graph ' + props.classNames} style={{paddingTop: 35}}>
```

Comment:
Ensures top of DAG is always under options menu:
![image](https://user-images.githubusercontent.com/614838/110347378-6c87a780-7fe5-11eb-8508-b414336debc7.png)